src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -82,6 +92,14 @@ export class CommentEditorComponent implements OnInit {
           event.preventDefault();
           this.insertOrRemoveCharsFromHighlightedText('_');
           break;
+        case 'z':
+          event.preventDefault();
+          this.undo();
+          break;
+        case 'y':
```

Comment:
Mac users do not use 'y' for redo

Reply:
I swapped to listening for the event `"historyUndo"` and `"historyRedo"`. This should allow us to standardize across different platforms

Reply:
Update: upon further testing, I think I will manually check for the undo and redo event as these events only fires when Javascript recognise that undo and redo is possible. If we call preventDefault(), redo action will no longer be callable.